### ISSUE TITLE
训练细节有点困惑，了解一下。

### ISSUE THREAD

**Issue by 786440445** [tencent-ailab/MuQ#9]
你们训练的seq_len大概是啥样的，我使用更大数据量在开源的模型上继续训练了4epoch，训练集合上ACC只有40-50左右。另外开源的模块好像没有recon_loss_ratio，训练这个是不需要加重构的mel_loss吗？

**Comment by juhayna-zh** [comment]
Our training used a fixed 30‑second audio segment length with a batch size of 192. Yes, the MuQ model presented in our paper does not require adding a reconstruction loss (e.g., mel loss).

Due to differences in implementation details, metrics may vary; however, an accuracy of only 40–50% does seem unusual. 
Since we’ve received many inquiries about training details, we have recently open‑sourced the training code. You can find it under [src/recipes](https://github.com/tencent-ailab/MuQ/tree/main/src/recipes).

Hope this helps, and wishing you smooth progress with your experiments.
